pkg/synthetictests/networking.go

```diff
@@ -16,6 +17,8 @@ type testCategorizer struct {
 	substring string
 }
 
+const ovnReadinessThreshold = 20
```

Comment:
how about a single threshold constant for both this and the original events failure.